src/server/db/models/github-user.js

```diff
@@ -0,0 +1,6 @@
+import db from '../index';
+
+export const GitHubUser = db.Model.extend({
+  tableName: 'github_user',
```

Comment:
Yeah, this is an IMO somewhat annoying design decision of Bookshelf.  I've added a block comment as a workaround.